icx/src/main.rs

```diff
@@ -399,6 +399,7 @@ async fn main() -> Result<()> {
                         result.unwrap_or(Err(AgentError::TimeoutWaitingForResponse()))
                     }
                     SubCommand::Query(_) => {
```

Comment:
FYI. I find some typos in error messages in this file when I am working on a demo.
`Got an error when make` should be `Got an error when making` .
`Got an error when send` should be `Got an error when sending` .

Reply:
I've been meaning to clean up icx for a while, there's lots that's wrong in here besides that. It'll be a separate PR though.